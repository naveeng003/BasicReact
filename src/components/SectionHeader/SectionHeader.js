import React from 'react';
import PropTypes from 'prop-types';
import styles from './SectionHeader.module.css';

const SectionHeader = props => {
  return (
    <h4 className="mb-3">{props.title}</h4>
  );
};

SectionHeader.defaultProps = {
  'title': 'No Title',
};

SectionHeader.propTypes = {
  'title': PropTypes.string.isRequired,
};

export default SectionHeader;
