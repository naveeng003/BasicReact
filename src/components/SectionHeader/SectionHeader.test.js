import React from 'react';
import SectionHeader from './SectionHeader';
import renderer from 'react-test-renderer';

describe('SectionHeader', () => {
   it('renders without error', () => {
      const tree = renderer
      .create(<SectionHeader title="test" />)
      .toJSON();
      expect(tree).toMatchSnapshot();
   });
});
