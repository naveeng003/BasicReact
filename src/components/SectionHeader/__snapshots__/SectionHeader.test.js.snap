// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader renders without error 1`] = `
<h4
  className="mb-3"
>
  test
</h4>
`;
